output_3.3.txt

```diff
@@ -0,0 +1,8 @@
+TASK: 3.3
+
+db.restaurants.updateMany( { "borough": "Manhattan" }, { $set: { grade: "A", score: 7, date: ISODate()  } });
```

Comment:
incorrect query
the structure of the documents is being broken - you set `grade`, `score` and `date` directly into the documents instead of their  `grades` array